scss/lessonCard.scss

```diff
@@ -1,18 +1,17 @@
 .lesson-card__title {
   font-size: 1.5rem;
-  font-weight: bold;
-  line-height: 1.33;
+  line-height: 1.3;
   margin-bottom: 0.75rem;
 }
-.lesson-card__icon-container {
-  display: inline-block;
-  margin-right: 40px;
-}
 .lesson-card__icon-text {
-  font-size: 0.8125rem;
-  line-height: 1.38;
+  font-size: 0.8rem;
+  line-height: 1.4;
 }
-.lesson-card__description {
-  font-size: 0.9375rem;
-  line-height: 1.6;
+.lesson-card__container_completed {
+  background-color: #00cf92;
```

Comment:
remove, We should be setting site-wide `green` color for our success badges. To do this, add into `index.scss`:

```
// available variables: https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss
$green: #00cf92 !default;
```
Make sure you add in the comment, took me awhile to find the variable sheet ;)